plugins/aws/iam/rootSigningCertificate.js

```diff
@@ -0,0 +1,60 @@
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Root Account Active Signing Certificates',
+    category: 'IAM',
+    description: 'Ensures the root user is not using x509 singing certificates',
+    more_info: 'AWS supports using x509 signing certificates for API access, but these should not be attached to the root user, which has full access to the account.',
+    link: 'https://docs.aws.amazon.com/whitepapers/latest/aws-overview-security-processes/x.509-certificates.html',
+    recommended_action: 'Delete the x509 certificates associated with the root account.',
+    apis: ['IAM:generateCredentialReport'],
+    compliance: {
+        hipaa: 'HIPAA requires strong auditing controls surrounding actions ' +
+                'taken in the environment. The root user lacks these controls ' +
+                'since it is not tied to a specific user. The root access keys ' +
+                'should not be used.'
+    },
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+
+        var region = helpers.defaultRegion(settings);
+
+        var generateCredentialReport = helpers.addSource(cache, source,
+            ['iam', 'generateCredentialReport', region]);
+        
+        if (!generateCredentialReport) return callback(null, results, source);
+
+        if (generateCredentialReport.err || !generateCredentialReport.data) {
+            helpers.addResult(results, 3,
+                'Unable to query for root user: ' + helpers.addError(generateCredentialReport));
+            return callback(null, results, source);
+        }
+
+        var found = false;
+        for (var r in generateCredentialReport.data) {
+            var obj = generateCredentialReport.data[r];
+            const resource = obj.arn;
+
+            if (obj && obj.user === '<root_account>') {
```

Comment:
check if user exists before doing the comparison.
Suggestion: `if (obj && obj.user && obj.user === '<root_account>') {`